db/structure.sql

```diff
@@ -2,16 +2,12 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.3
```

Comment:
nice!
I guess any PostgreSQL's configuration is different from this `structure.sql`.
I'm happy unnecessary diffs aren't there.